conjure-java-core/src/test/java/com/palantir/conjure/java/UndertowEteResource.java

```diff
@@ -178,6 +197,109 @@ public String jsonErrorsHeader(AuthHeader authHeader, String headerParameter) {
         throw ConjureErrors.invalidServiceDefinition("my-service-string", Optional.of(SimpleEnum.VALUE));
     }
 
+    @Override
+    public String errorParameterSerialization(AuthHeader authHeader, String headerParameter) {
+        if (headerParameter.startsWith("JSON") || headerParameter.equals("TOSTRING")) {
```

Comment:
Any reason not to always throw?

Reply:
I wanted to also add a test to ensure that we're able to deserialize the return type correctly.